openstack/fixtures/helpers.go

```diff
@@ -129,13 +129,73 @@ func handleCreateUser(t *testing.T, w http.ResponseWriter, r *http.Request, user
         "links": {
             "self": "https://example.com/identity/v3/users/%[1]s"
         },
-        "name": "James Doe",
+        "name": "James_Doe",
```

Comment:
Why?

Reply:
There was a lot of debugging ongoing but this change doesn't really influence anything, so no point in that.